configuration/locations/core-locations.csv

```diff
@@ -0,0 +1,2 @@
+UUID,Void/Retire,Name,Description
+AF939782-898B-409A-99A4-D1653484EDBD,,Patient Home,The patient is visited at home
```

Comment:
What about "Unknown Location"? Are we assuming that is part of the core OpenMRS install?

Reply:
I would assume so, but doesn't hurt to include it for completeness.  I'd vote to include it here.

Reply:
Initially I had it included and it caused an NPE in concept validation, something about its location attributes being null. Chose not to worry about it.

Reply:
Strange... what was the error @brandones

Reply:
Huh, it worked this time when I re-ran it. Added that line back. We'll see if it causes trouble.